tests/cephfs/cephfs_utilsV1.py

```diff
@@ -999,3 +999,22 @@ def subvolume_authorize(self, client, vol_name, subvol_name, client_name, **kwar
         if kwargs.get("extra_params"):
             command += f" {kwargs.get('extra_params')}"
             client.exec_command(sudo=True, cmd=command)
+
+    def heartbeat_map(self, mds):
+        """
+        Verify "heartbeat_map" timeout is not in mds log
+        Args:
+            mds: mds node
+        """
+        try:
+            mds.exec_command(
+                sudo=True,
+                cmd="cat /var/log/ceph/ceph-mds.%s.log "
+                "| grep heartbeat_map" % mds.node.shortname,
+            )
```

Comment:
Opt to use `format strings` like `cmd=f"grep heartbeat_map /var/log/ceph/ceph-mds.{mds.node.shortname}.log"`